src/User.php

```diff
@@ -59,6 +59,11 @@ public static function matchContact(UserInterface $user, $prefix = '') {
     /* @var \Drupal\civiremote\CiviMRF $cmrf */
     $cmrf = Drupal::service('civiremote.cmrf');
     $config = Drupal::config('civiremote.settings');
+
+    // Only match locked users when configured.
+    if ($user->isBlocked() && !($config->get('match_locked_users') ?? FALSE)) {
```

Comment:
Looking at this line I would expect the setting key to be `match_blocked_users`. Is there a reason that it uses `locked` instead of `blocked`?

Reply:
You're right, this should really be *blocked*, as that's the Drupal term.